doc/manual/types.rst

```diff
@@ -1317,13 +1313,13 @@ To construct an object representing a missing value of type ``T``, use the
 .. doctest::
 
     julia> x1 = Nullable{Int64}()
-    Nullable{Int64}()
+    #NULL
```

Comment:
When https://github.com/JuliaLang/julia/pull/16354 will be merged, the output will go back to the old one. This was just a temporary quirk due to `limit_output(STDOUT)` being `true`.